README.md

```diff
@@ -108,22 +108,26 @@ $ make && ctest
 
 Celeritas guarantees full compatibility and correctness only on the
 combinations of compilers and dependencies tested under continuous integration:
-- Compilers:
-    - GCC 8.4, 12.3
-    - Clang 10.0, 15.0
-    - GCC 11.3 + NVCC 11.8
-    - HIP-Clang 15.0
+- Compilers and standard:
+    - GCC 8, 11, 12, 14
+    - Clang 10, 15, 18
+    - GCC 11.5 + NVCC 12.6
+    - ROCm Clang 18
+    - C++17 and C++20
 - Dependencies:
     - Geant4 11.0.3
-    - VecGeom 1.2.5
+    - VecGeom 1.2.10
 
 Partial compatibility and correctness is available for an extended range of
 Geant4:
 - 10.5-10.7: no support for tracking manager offload
 - 11.0: no support for fast simulation offload
-- 11.1-11.2: [no support for default Rayleigh scattering cross section](see
+- 11.1-11.3: [no support for default Rayleigh scattering cross section](see
   https://github.com/celeritas-project/celeritas/issues/1091)
 
+Note also that navigation bugs in older versions of Geant4 and VecGeom can
```

Comment:
Do we know the upper bound version number of those bugs (i.e. version number after which those bugs are all gone)?

Reply:
I think "bug-free" is an unbounded function for any code 😄 but 11.3 is the first one with no known bugs (twisted object navigation being some of the more recent issues).

Reply:
```suggestion
Note also that known navigation bugs in versions of Geant4 older than v11.3 and VecGeom v1.x will cause test failures in Celeritas.
```